### PR TITLE
done basic database function

### PR DESCRIPTION
**Pull request by davidcoh464**

### CODE REVIEW

File: DataBase/database_interface.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+from typing import Dict, Any, Optional
+
+import firebase_admin
+from firebase_admin import credentials, db
+
+cred = credentials.Certificate("serviceAccountKey.json")
```
Comment (axshai) [inline DataBase/database_interface.py:7]: it should be a const, no?

File: DataBase/serviceAccountKey.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (axshai) [inline DataBase/serviceAccountKey.json]: please dont push private keys

File: DataBase/database_interface.py (hunk 1 above)
Comment (davidcoh464) [reply DataBase/database_interface.py:7]: This is the convention as suggested in firebase

File: DataBase/serviceAccountKey.json (hunk 2 above)
Comment (davidcoh464) [reply DataBase/serviceAccountKey.json]: file was removed

**Pull request merged**